internal/helm/helm.go

```diff
@@ -310,79 +320,16 @@ func filterVariables(s map[string]interface{}, variables []string, prefix string
 	}
 }
 
+// unquote unquotes gotemplates in template
+func unquote(template string) string {
+	re := regexp.MustCompile(`'(\{\{[^'}]*\}\})'`)
+	return re.ReplaceAllString(template, "$1")
```

Comment:
The only caller is `stackData = []byte(unquote(string(stackData)))` so could this not take/give a `[]byte` and have:
`return re.ReplaceAll(template, []byte("$1"))` for less casting overall?